sdk/python/feast/data_source.py

```diff
@@ -634,6 +644,7 @@ class PushSource(DataSource):
 
     def __init__(
         self,
+        *,
         name: str,
         schema: Dict[str, ValueType],
         batch_source: DataSource,
```

Comment:
SInce we're adding the * for keyword args, should we change these to keyword args too then aka Optional?

Reply:
They become keyword args, but are not optional (also this is a safe change since Push sources haven't been released yet)